### DISCUSSION TITLE
Can I write things below the note without syncing it ?

### DISCUSSION THREAD

**Discussion by SiddharthManthan** [windingwind/zotero-better-notes#1019] [Q&A] [answered]
I could not find it in documentation. Can I write things below the synced note without syncing it back to the pdf ?

eg : 
> Note 1
- Some information

> Note 2

I do not want to sync "Some Information"

**Comment by windingwind** [comment, marked as answer]
Once the annotation is embedded/converted to a note, any modification won't affect the original one.
